tesseract_time_parameterization/src/time_optimal_trajectory_generation.cpp

```diff
@@ -845,7 +863,7 @@ double Trajectory::getMinMaxPathAcceleration(double path_position, double path_v
   double max_path_acceleration = std::numeric_limits<double>::max();
   for (unsigned int i = 0; i < joint_num_; ++i)
   {
-    if (config_deriv[i] != 0.0)
+    if (!tesseract_common::almostEqualRelativeAndAbs(config_deriv[i], 0.0, std::numeric_limits<double>::epsilon()))
```

Comment:
This change seems to make the trajectory duration increase for the tests. It was dividing by very small numbers because they never actually equaled zero.